.circleci/config.yml

```diff
@@ -1,8 +1,3 @@
-#
-# Copyright (c) 2021, Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Why header removed from here?

Reply:
thought it was overkill for a config file, but added it back